Testscripts/Linux/nvme_blkdiscard.sh

```diff
@@ -17,8 +17,15 @@
     exit 0
 }
 UtilsInit
+
+if [ "$DISTRO" == "ubuntu_14.04" ]; then
```

Comment:
I see Azure Gallery shows 7 different revisions. Please make sure this condition can accept all revision's $DISTRO value.

Reply:
This covers all 14.04 dot releases, the test results are for 14.04.6/14.0.4.5 for example.